### ISSUE TITLE
UTD errors are shown incorrectly in room previews

### ISSUE THREAD

**Issue by HarHarLinks** [element-hq/element-meta#2462]
### Steps to reproduce

1. enable knocking in labs
2. enable message preview in room list
3. knock on room
4. get rejected

#### What did you expect?
"Alice rejected your knock"
#### What happened instead?
UTD

### Operating system

arch

### Application version

Element Nightly version: 2024061201 Crypto version: Rust SDK 0.7.0 (068a0af), Vodozemac 0.6.0

### How did you install the app?

aur

### Homeserver

synapse 1.106

### Will you send logs?

Yes

**Comment by richvdh** [comment]
No, the issue as reported "UTD errors are shown incorrectly in room previews" is a web bug.
